src/obis-observer/commands/downlink/getArchiveState.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Downlink command.
```

Comment:
would be nice to copy-paste commands description from jooby-docs

here and in other places

Reply:
done